src/admin/admin.testData.ts

```diff
@@ -33,3 +33,16 @@ export const mockAdminData = [
     isDeleted: true,
   },
 ];
+
+export const mockAdminDataWithPermission = {
+  id: "634818ff801e2c37130565f5",
+  email: "admin@gmail.com",
+  permission: "normal",
```

Comment:
what means withPermission? it's super admin or all types of admin?

Reply:
It just means a mock data with permission field.